core/src/main/resources/reference.conf

```diff
@@ -29,4 +29,9 @@ health-monitor-dispatcher {
 
 gcs {
   bucketLogsMaxAge = "180"
+}
+
+submissionmonitor {
+  submissionPollInterval = 1m
```

Comment:
That's what it was using before, it was just set via a default parameter in code rather than a config file:
https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala#L24